src/movieInfo/movieInfo.service.js

```diff
@@ -0,0 +1,19 @@
+(function (angular) {
+    'use strict';
+
+    angular
+        .module('movieClub')
+        .factory('movieInfo', movieInfo);
+
+    function movieInfo($http) {
+        var factory = {
+            get: get
+        };
+        return factory;
+
+        function get(movieName) {
+            return $http.get('http://www.omdbapi.com/?t=' + movieName + '&tomatoes=true&plot=full');
```

Comment:
Do you mind if we remove the protocol from the URL here so that it uses whatever the active protocol is? (e.g. `$http.get('//www.omdbapi.com/...');`)